README.md

```diff
@@ -133,7 +134,36 @@ The following example configures Fast DDS to publish synchronously, and to have
                 <historyMemoryPolicy>PREALLOCATED_WITH_REALLOC</historyMemoryPolicy>
             </publisher>
 
-            <subscriber profile_name="subscriber profile" is_default_profile="true">
+            <!-- Publisher profile for topic helloworld -->
+            <publisher profile_name="helloworld">
+                <qos>
+                    <publishMode>
+                        <kind>SYNCHRONOUS</kind>
+                    </publishMode>
+                </qos>
+            </publisher>
+
+            <!-- Request subscriber profile for services -->
+            <subscriber profile_name="service">
+                <historyMemoryPolicy>PREALLOCATED_WITH_REALLOC</historyMemoryPolicy>
+            </subscriber>
+
+            <!-- Request publisher profile for clients -->
+            <publisher profile_name="client">
+                <qos>
+                    <publishMode>
+                        <kind>ASYNCHRONOUS</kind>
+                    </publishMode>
+                </qos>
+            </publisher>
+
+            <!-- Request subscriber profile for server of service "add_two_ints" -->
+            <subscriber profile_name="rq/add_two_intsRequest">
+                <historyMemoryPolicy>PREALLOCATED_WITH_REALLOC</historyMemoryPolicy>
+            </subscriber>
+
+            <!-- Reply subscriber profile for client of service "add_two_ints" -->
+            <subscriber profile_name="rr/add_two_intsReply">
```

Comment:
@EduPonz hmm, these README changes look like a follow-up after #335 and unrelated to this patch. Consider putting them in a separate PR.

Reply:
Done

Reply:
README changes have been moved to a separate PR #504